kie-server-parent/kie-server-services/kie-server-services-openshift/src/main/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateCloudRepository.java

```diff
@@ -22,19 +22,26 @@
 import org.kie.server.api.model.KieServerConfigItem;
 import org.kie.server.services.impl.storage.KieServerState;
 import org.kie.server.services.impl.storage.KieServerStateRepository;
-import org.kie.server.services.openshift.api.KieServerReadinessProbe;
 import org.kie.soup.commons.xstream.XStreamUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class KieServerStateCloudRepository implements KieServerStateRepository,
-                                                    KieServerReadinessProbe, CloudClientFactory {
+public abstract class KieServerStateCloudRepository implements KieServerStateRepository, CloudClientFactory {
 
     public static final String ROLLOUT_REQUIRED = "services.server.kie.org/openshift-startup-strategy.rolloutRequired";
     public static final String CFG_MAP_DATA_KEY = "kie-server-state";
     public static final String CFG_MAP_LABEL_NAME = "services.server.kie.org/kie-server-state";
     public static final String CFG_MAP_LABEL_VALUE_USED = "USED";
     public static final String CFG_MAP_LABEL_VALUE_IMMUTABLE = "IMMUTABLE";
+    public static final String CFG_MAP_LABEL_APP_NAME = "application";
+    public static final String ENV_HOSTNAME = "HOSTNAME";
+    
+    /**
+     *  To be refactored into org.kie.workbench.common.screens.server.management.utils.ControllerUtils class
+     *  by separate PR
+     */
+    public static final String CFG_KIE_CONTROLLER_OCP_GLOBAL_DISCOVERY_ENABLED = 
+            "org.kie.workbench.controller.openshift.global.discovery.enabled";
```

Comment:
I'd remove the workbench from the name and stick with org.kie.controller....

Reply:
Update constant value as per suggested and move to KieServerControllerConstants class